my-app/src/Components/header/header.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './header.scss';
+
+const Header = () => {
+  return (
+    <header>
+
```

Comment:
is a need here free space?

Reply:
@NikitaHapeyeu This component is redundant. Deleted